x/staking/src/keeper/traits.rs

```diff
@@ -0,0 +1,170 @@
+use gears::types::account::{Account, ModuleAccount};
+
+pub use super::*;
+
+/// AccountKeeper defines the expected account keeper methods (noalias)
+// TODO: AuthKeeper should implements module account stuff
+pub trait AccountKeeper<SK: StoreKey>: AuthKeeper<SK> + Clone + Send + Sync + 'static {
+    // TODO: should be a sdk account interface
+    fn get_account<DB: Database, CTX: QueryableContext<DB, SK>>(
+        &self,
+        ctx: CTX,
+        addr: ValAddress,
+    ) -> Account;
+
+    fn get_module_account<DB: Database, CTX: QueryableContext<DB, SK>>(
+        &self,
+        ctx: &CTX,
+        module_name: String,
+    ) -> Option<ModuleAccount>;
+
+    fn set_module_account<DB: Database, CTX: TransactionalContext<DB, SK>>(
+        &self,
+        context: &mut CTX,
+        acc: ModuleAccount,
+    );
+}
+
+/// BankKeeper defines the expected interface needed to retrieve account balances.
```

Comment:
In future we can move this into Gears... it's OK here for now.

Reply:
The staking module in sdk owns this trait. 
We can move it or just use the from the module.